docs/src/pages/css-in-js/basics/AdaptingHook.tsx

```diff
@@ -9,22 +9,22 @@ interface Props {
 }
 
 const useStyles = makeStyles({
-  root: {
-    background: (props: Props) =>
+  root: (props: Props) => ({
+    background:
       props.color === 'red'
         ? 'linear-gradient(45deg, #FE6B8B 30%, #FF8E53 90%)'
         : 'linear-gradient(45deg, #2196F3 30%, #21CBF3 90%)',
     border: 0,
     borderRadius: 3,
-    boxShadow: (props: Props) =>
```

Comment:
Why changing it?

Reply:
There are no types defined for setting CSS properties to be a function that returns a string

Reply:
It's possible to do it for some of them, looking into the others, will report back

Reply:
@oliviertassinari I've figured it out, will revert this and implement the new type definition